Base CONTRIBUTING.md which points to the wiki (#41)
terrapkg/packages: CONTRIBUTING.md

# Contributing

> First of all, thanks for wanting to contribute to Terra! If you have any questions about contributing, please contact us on [Discord](https://discord.gg/5fdPuxTg5Q). Since we use a GitHub Wiki for [documention](https://github.com/terrapkg/packages/wiki), our guidelines may be found there. See the links below:

Reviewer: ```suggestion
First of all, thanks for being interested in contributing to Terra! If you have any questions about contributing, please contact us on [Discord](https://discord.gg/5fdPuxTg5Q). Since we use a GitHub Wiki for [documention](https://github.com/terrapkg/packages/wiki), our guidelines may be found there. See the links below:
```

* [Contribution Guide](https://github.com/terrapkg/packages/wiki/Contribute)
* [FAQ](https://github.com/terrapkg/packages/wiki/FAQ)

> <!-- TODO: Submission/Package Policy -->

Reviewer: ```suggestion
* [Policy](https://github.com/terrapkg/packages/wiki/Policy)
```
